### PR TITLE
8: UI creates layout template

### PR DESCRIPTION
**Pull request by jtroussard**
Adds,

- hello world home 
- layout template
- active page feature for navigation bar
- interpolate username in navigation bar
- some styling for username

### CODE REVIEW

File: templates/home.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{% extends 'layout.html' %}
+
+{% block styles %}
+<style>
+    /* Style for active page */
+    .nav-item.active {
```
Comment (JPC1505) [inline templates/home.html:6]: move this into a css static file

File: tests/test_app.py [hunk 2]
```diff
@@ -1,10 +1,32 @@
...
+
+def test_home_page_active_nav(client):
+    """Test that the home page has the active class on the correct nav item."""
+    response = client.get('/')
+    soup = BeautifulSoup(response.data, 'html.parser')
+    home_li = soup.find('a', string="Home").find_parent('li')
+    assert home_li is not None
+    assert 'active' in home_li['class']
+
+def test_home_page_login_name(client, monkeypatch, app):
```
Comment (JPC1505) [inline tests/test_app.py:27]: Figure out how to access app context and mock route variable login_name
test_home_page_login_name

**Review by JPC1505 — CHANGES_REQUESTED** [review]

File: templates/home.html (hunk 1 above)
Comment (jtroussard) [reply templates/home.html:6]: created td issue

File: tests/test_app.py (hunk 2 above)
Comment (jtroussard) [reply tests/test_app.py:27]: created td issue

**Review by JPC1505 — APPROVED** [review]
lgtm

**Pull request merged**